book/07-git-tools/sections/advanced-merging.asc

```diff
@@ -388,25 +388,25 @@ end
 hello()
 ----
 
-If you like this format, you can set it as the default for future merge conflicts by setting the `merge.conflictstyle` setting to `diff3`.
+Kung gusto mo ang format na ito, maaari kang magtakda nito bilang default para sa hinaharap na merge na mga salungat sa pamamagitan ng pagtatakda ng `merge.conflictstyle` at pagtatakda sa `diff3`.
 
 [source,console]
 ----
 $ git config --global merge.conflictstyle diff3
 ----
 
-The `git checkout` command can also take `--ours` and `--theirs` options, which can be a really fast way of just choosing either one side or the other without merging things at all.
+Ang `git checkout` na utos ay maaari ding kumuha ng `--ours` at `--theirs` na mga opsyon, na kung saan ay maaaring maging isang tunay na mabilis na paraan sa pagpili pa lamang alinman sa isang panig o sa iba na hindi naka merge sa mga bagay sa lahat.
 
-This can be particularly useful for conflicts of binary files where you can simply choose one side, or where you only want to merge certain files in from another branch - you can do the merge and then checkout certain files from one side or the other before committing.
+Ito ay maaaring maging partikular na kapaki-pakinabang para sa mga salungat na mga binary file na kung saan maaari mo lamang pumili ng isang panig, o kung saan gusto lamang i-merge ang tiyak na mga file mula sa ibang branch - maaari mo ding gawin ang pag-merge at pagkatapos i-checkout tiyak na mga file mula sa isang panig o sa iba bago mag-commit.
```

Comment:
> maaari mo lamang pumili ng isang panig
to
> maaari mo lamang pillin ang isang panig
or
> maaari ka lamang pumili ng isang panig

Reply:
done change, using maaari ka lamang pumili ng isang panig